guides/doc-Managing_Content/master.adoc

```diff
@@ -55,9 +55,6 @@ ifdef::satellite,katello[]
 include::common/assembly_importing-kickstart-repositories.adoc[leveloffset=+1]
 endif::[]
 
-[appendix]
-include::common/modules/proc_importing-content-isos-into-a-connected-foreman.adoc[leveloffset=+1]
```

Comment:
Please also delete the file itself and check that all anchors within this file are not used anywhere throughout the docs.

Reply:
Not sure why build is failing. I have checked and there is no anchor which has used this as a reference.
Any idea what I am missing here @maximiliankolb ?

Reply:
This was most likely just a temporary network fluke. I've restarted the failed GHA.

Reply:
It is still failing!